### PR TITLE
Rs 2022 04 03.1

### PR DESCRIPTION
**Pull request by rschlaikjer**
burgertime-cover cover_large

### CODE REVIEW

**Comment by rschlaikjer** [comment]
Note that I've edited previous instances of 半人前 - we had this as 'half-vampire' but I don't think that's really correct, both because
1. It implies she's half something _else_ and
2. I think it's more supposed to be 'incomplete', 'unfinished', someone that has not consummated their vampireness

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 1]
```diff
@@ -41,12 +46,13 @@ After intensely studying the menu, in the end, she went for the same set menu th
...
 -- 席に座って、アルクェイドと向かい合う。
-I take a seat and face Arcueid.
+I take a seat across from Arcueid.
 }
 [sha:d4a30505506945451f2e8e30f000e6fe1ccf199b]{
 -- Page 297, Offset 25012.
 -- アルクェイドはきょろきょろと周囲を見渡したあと、慣れた手つきでフライドポテトを口に運んだ。
-After restlessly looking around at her surroundings, she brings a fry to her mouth in a manner that shows her familiarity with such food.
+After nervously glancing around at her surroundings,
+ she brings a fry to her mouth in a manner that demonstrates familiarity with such food.
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:55]: suggests familiarity

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 2]
```diff
@@ -72,17 +78,21 @@ I thought for sure this would be your first time in a place like this."
 [sha:956936902c5513fda9e448fb010ed8c981cf25c5]{
 -- Page 301, Offset 25017.
 -- 西洋の吸血鬼のイメージと、ヴローヴの服装があまりにも時代がかっていたので、勝手にゴシックな印象を抱いていた。吸血鬼社会とやらもしっかり近代化が進んでいるのかもしれない。
-The traditional western image of a vampire combined with Vlov fitting into that old-fashioned look had given me the impression that vampires are gothic creatures. Maybe even vampire society has kept up with the modern world.
+The tradition Western image of a vampire, coupled with Vlov's anachronistic clothing choices,
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:81]: traditional

Is "anachronistic clothing choices" really more natural than "old-fashioned look"?

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 3]
```diff
@@ -72,17 +78,21 @@ I thought for sure this would be your first time in a place like this."
...
+ had left me with a very Gothic-inspired mental model.
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:82]: not sure I agree with "Gothic-inspired mental model". I say revert

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 4]
```diff
@@ -152,9 +171,10 @@ Even though I knew that people gathered here like this, I'd never experienced it
 [sha:b2478d18b719c22dc758998adf58c4ebf2bbd585]{
 -- Page 307, Offset 25033.
 -- 情報だけの世界はそれはそれで<真|しん>だけど、受肉からの感覚は正しい理論より芯がある。
-// Sorry, what?
-// I'm not sure how to work in the しん pun here...
-A world of only information would itself be 'truth'. However, the sensation stemming from incarnation is a core deeper than any correct theory.
+// Kinda slid around which word triggers the しん/しん, not sure I love it still
+A world built only of information is 'real' in and of itself,
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:175]: I think this is a pretty good solution. Well, it doesn't really have the same match but I think it's good

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 5]
```diff
@@ -164,7 +184,9 @@ This must be what they mean when they say experience outweighs theory.
 [sha:d3c2049bb5f785cff6e066f8b9c9642ac6ef0f84]{
 -- Page 307, Offset 25035.
 -- 何千億の単語を覚えていても、実際にやってみないと何も手に入らないんだなって」
-No matter how many billions of words you learn, you can't get anything if you don't give it a go yourself."
+// Anki crunchers malding
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:187]: @gambsvn

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 6]
```diff
@@ -199,18 +223,22 @@ Arcueid's facial expression grows darker and darker.
 [sha:58b71adc544e3d3bdfae244d9408713072ba453e]{
 -- Page 310, Offset 25042. Glued. Mods: @x.
 -- アルクェイドにつられてか、俺も気持ちが沈んでしまった。うまく言えないが、こいつのそういう顔を見るのはイヤだった。
-Perhaps it's because I'm drawn to Arcueid, but even I'm starting to feel depressed. I can't put it into words, but seeing her face like that makes me feel bad.
+I must've got caught up in the moment with Arcueid. I'm starting to feel depressed as well.
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:226]: depressed myself.

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 7]
```diff
@@ -199,18 +223,22 @@ Arcueid's facial expression grows darker and darker.
...
+ I don't feel like I could eloquently express %{i}why%{/i}, but I just can't bear seeing her face like this.
 }
 [sha:6187b5f15e1df274a5699c35836434597a58c0bf]{
 -- Page 311, Offset 25043.
 -- 「早計じゃないか？　たんにさ、理論だけで経験を補えるヤツと、経験で理論を覆せるヤツがいるだけだと思うけど？
-"Isn't that conclusion bit premature, though? I think it's simply that there are those who make up for lack of experience with theory alone, and those who overturn theory with their experience.
+"Isn't that conclusion a bit premature, though?
+ I think what it all boils down to is that there are some people who make up for a lack of experience with theory alone,
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:233]: I think that what it all boils down to is that there are people who can make up for their lack of experience with theory alone, and those who can overturn theory through the benefit of their experiences.

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 8]
```diff
@@ -398,17 +445,18 @@ The mysterious beauty from before has slightly faded away, and now the scene has
 [sha:9bee9e337b4b1ed8ba08ac29dacdbbec0a22e1c4]{
 -- Page 327, Offset 25081.
 -- 食事をし、活動に必要なだけの栄養を摂らなければ生き物は生きてはいけない。
-It's a form of nutrition, and if a living being doesn't get enough to perform it's activities at the very least, then it wouldn't be able to survive.
+If a creature doesn't eat and receive the nutrients it needs to function,
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:448]: doesn't/wouldn't mismatch

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 9]
```diff
@@ -458,42 +507,47 @@ I mean,
...
 -- Page 333, Offset 25095.
 -- 「あのね、志貴。基本的にわたしは食事を摂らないんだ。
-"Listen, Shiki. Normally I wouldn't be eating a meal.
+"Listen, Shiki. Eating meals isn't something I do all that often.
 }
 [sha:6f651c917bfb75e50bd8012a975e6732d481cf8b]{
 -- Page 333, Offset 25096.
 -- 確かに、こういうふうに栄養素を口にすれば自分の力だけで動けるけど、それは気分の問題なの」
-It's true that if I consume nutrients like this, then I can operate on my own strength alone, but that's just a matter of preference."
+It's true that consuming nutrients like this allows me to move under my own steam,
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:527]: "move under my own steam" is kinda awkward. Probably just go back to "strength"

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 10]
```diff
@@ -458,42 +507,47 @@ I mean,
...
+ but whether I do so or not is simply a mater of preference."
 }
 [sha:5d5fb2cee3390074e49b22b0f4eecf71773c3f4c]{
 -- Page 334, Offset 25097.
 -- 「わたしの熱量の補充方法は志貴たちとは違う。たしかに食欲はあるけど、それは人間で言うところの性欲と同じかな。
-"My method of replenishing calories is different than yours. Sure, I have an appetite, but it's more like the so-called sex drive in human beings.
+"My method of replenishing calories is different than yours.
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:533]: different from yours

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 11]
```diff
@@ -458,42 +507,47 @@ I mean,
...
 -- 食べないとイライラするけど、わたしはあんまり食事に重きを置いていないから、反転衝動に襲われる事は少ないよ」
-I get irritated when I don't eat, but I don't place a lot of importance on food, so I don't get attacked by inversion impulses that much."
+I get irritated when I don't eat, sure,
+ but at the same time I don't place a lot of importance on food so that I don't get attacked by inversion impulses as often."
 }
 [sha:e3f50ecdba7d6814dd92d2daba54ca4a909b81dc]{
 -- Page 335, Offset 25099.
 -- あっさりとアルクェイドは血液食事説を否定した。
-Just like that, Arcueid shuts down the blood diet theory.
+Arcueid cleanly overturns my blood diet theory.
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:545]: not a fan of "cleanly overturns". repudiates? refutes?

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 12]
```diff
@@ -458,42 +507,47 @@ I mean,
...
 }
 [sha:899e715212295898dd70ba8a6264a854221d63e6]{
 -- Page 335, Offset 25100.
 -- 彼女は本当に、血を飲まなくてもいいらしい。
-It really does seem like she's fine without ever drinking blood.
+It seems like she really can get by just fine without drinking any blood.
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:550]: "can" italics

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 13]
```diff
@@ -635,13 +694,16 @@ However―――
 [sha:dd2d30ebc879f20af0bbe9ede7d32fa1037636c2]{
 -- Page 345, Offset 25128.
 -- <人間|どうぶつ>は娯楽がなければ生きていけない、と言われているようなもので、人間として認めがたいというか……。
-It's like being told that humans can't live without pleasure, and as a human being, I find that hard to believe...
+// Hedonism is the only real philosophy, fight me
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:697]: image

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 14]
```diff
@@ -686,17 +749,20 @@ But you said that you hate sucking blood, right?
...
 -- Page 349, Offset 25139.
 -- 会心の閃きを口にする。
-I blurt out the satisfying revelation.
+I blurt out my satisfying revelation.
 }
 [sha:019643faeef74e7ec3ccb1725d3b703bca8cee2b]{
 -- Page 349, Offset 25140.
 -- しかし、アルクェイドは辛そうに眉をひそめた。
-However, Arcueid frowns as if she is in pain.
+// Kinda hate 辛い
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:764]: 辛い is the 困る of adjectives.

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 15]
```diff
@@ -836,13 +913,13 @@ I don't like what-ifs.
...
-"Is that so? But I like talking them.
+"Really? I kinda like them.
 }
 [sha:e2e7d23a162cb0375e3adf2945f3a23d1c9f98b4]{
 -- Page 357, Offset 25169.
 -- どんな結果になるか分からないけど、とりあえずその時は、どこかに救いがあるような気がするでしょう？」
-// what?
-I don't know what might happen as a result, but in the moment, it feels like salvation is coming from somewhere, right?"
+You never know where they might lead,
+ and when you're pursuing them don't you always feel like there's some grand truth out there waiting to be discovered?"
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:922]: Although I think this interpretation is right, the resulting grammar is wrong.
> You might not know where you're gonna end up, but in the moment you always feel like there's some grand truth out there, right?

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 16]
```diff
@@ -1169,17 +1264,18 @@ There is no sign of her hiding the truth.
 [sha:78420500694f908d3c1fd896a14a8e39ecd9cde0]{
 -- Page 383, Offset 25234.
 -- 「アイツの名前はミハイル。ミハイル・ロア・バルダムヨォン。人間から吸血鬼になった、どこにでもいる死徒の一人よ」
-"His name is Michael. Michael Roa Valdamjong. He is a human turned vampire, a single Dead Apostle that could be anywhere."
+"His name is Michael. Michael Roa Valdamjong.
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:1267]: image

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 17]
```diff
@@ -1279,7 +1380,9 @@ I was about to say 'lent you a hand' but stopped half-way.
 [sha:a84e4060b86c58ddd318d1166145cb3056b99b1b]{
 -- Page 392, Offset 25256.
 -- それは街を守るという理由を、違うものにする言葉だったからだ。
-It's because that reason was different from my stated reason of protecting the city.
+// Kinda reified, perhaps too much so
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:1383]: meh, i think its fine.

**Review by Avuxo — APPROVED** [review]
otsukare

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 18]
```diff
@@ -16,22 +19,24 @@ It's past 2:00PM now, so we decide to have a late lunch.
 [sha:e42961532c134863158087bdd9d9efc489a415b9]{
 -- Page 295, Offset 25006. Mods: @n.
 -- などと言うものだから、
-That is what she said———
+Arcueid has also voiced a preference,
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Arcueid/Day 10/10_00_ARC10_4.txt:22]: Yeah, I know. For some reason it just felt off to me with the staccato lines combined with the way the scene animations played. I definitely wouldn't put back the dashes though, these are the same sentence just spliced by a cg transition. I also wanted to include the devil's 仕方なく at least in spirit. 
> Arcueid says she wants something ordinary anyway,
> so I suck it up and take us to the food court.

File: script/Arcueid/Day 10/10_00_ARC10_4.txt (hunk 2 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 10/10_00_ARC10_4.txt]: I think to _me_ I'd say yeah, at least within the structure of this sentence. 'Fitting into a look' just doesn't click.

File: script/Arcueid/Day 10/10_00_ARC10_4.txt (hunk 8 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 10/10_00_ARC10_4.txt:448]: As in s/wouldn't/won't?

File: script/Arcueid/Day 10/10_00_ARC10_4.txt (hunk 16 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 10/10_00_ARC10_4.txt:1267]: Hello Michael

File: script/Arcueid/Day 10/10_00_ARC10_4.txt (hunk 18 above)
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Arcueid/Day 10/10_00_ARC10_4.txt:22]: this version is fine by me

File: script/Arcueid/Day 10/10_00_ARC10_4.txt (hunk 2 above)
Comment (Avuxo) [reply script/Arcueid/Day 10/10_00_ARC10_4.txt]: I can't say I agree. sounds perfectly natural to my eye's ears

**Pull request merged**